config.yaml

```diff
@@ -1,14 +1,41 @@
 options:
-  string-option:
+  github-client:
```

Comment:
Do we have intend to have so tight integration with github and google? How would we extend this? Add config variables here and update the template?